### PR TITLE
only focused input field submits to the form

### PR DESCRIPTION
**Pull request by AMBacelar**
## Who is this for?
users of the platform that want to jump to a page

## What is it doing for them?
not break the form by sending multiple page fields to the form

### CODE REVIEW

File: common/views/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -148,15 +149,17 @@ export const Pagination: FunctionComponent<Props> = ({
...
           <span aria-hidden>Showing page</span>
           <span id="searchInputLabel" className="visually-hidden">
             {`Showing page ${currentPage} / ${formatNumber(totalPages)}`}
           </span>
           <PageSelectorInput
             name="page"
-            form="search-page-form"
+            onFocus={() => setIsFocused(true)}
+            onBlur={() => setIsFocused(false)}
+            form={`${isFocused ? 'search-page-form' : ''}`}
```
Comment (rcantin-w) [inline common/views/components/Pagination/Pagination.tsx:162]:
```suggestion
            form={isFocused ? 'search-page-form' : ''}
```

(nitpick)

**Review by rcantin-w — COMMENTED** [review]
Are the UI changes expected, we want to display "Showing page"? (Cardigan awaiting review)

File: common/views/components/Pagination/Pagination.tsx [hunk 2]
```diff
@@ -148,15 +149,17 @@ export const Pagination: FunctionComponent<Props> = ({
         </Link>
       )}
 
-      {isEnhanced && isHiddenMobile ? (
```
Comment (rcantin-w) [inline common/views/components/Pagination/Pagination.tsx:151]: Ahh I think this is what is causing the UI change?

**Review by rcantin-w — COMMENTED** [review]
Seems to be working alright, yeah? 🤞 Works well locally!

File: common/views/components/Pagination/Pagination.tsx (hunk 2 above)
Comment (AMBacelar) [reply common/views/components/Pagination/Pagination.tsx:151]: yes, the jump to page should always be visible where the pagination component is used

**Review by rcantin-w — APPROVED** [review]

**Pull request merged**